apps/climatemappedafrica/src/pages/[[...slugs]].js

```diff
@@ -23,7 +25,7 @@ function Index({ blocks, fallback, ...props }) {
     pageConfigProps = { value: { fallback } };
   }
   return (
-    <Page {...props}>
+    <Page {...pageConfigProps}>
       <PageConfig {...pageConfigProps}>
```

Comment:
`pageConfigProps` sounds like something that should go into `PageConfig` rather than `Page`. Are we naming them wrong?